Form-Controls/styles.css

```diff
@@ -0,0 +1,267 @@
+/* #region Reset styles */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+button {
+    padding: 0;
+    font-family: inherit;
+    background: none;
+    border: none;
+}
+
+input,
+button,
+textarea,
+select {
+    margin: 0;
+    padding: 0;
+    font-family: inherit;
+    font-size: inherit;
+}
+
+/* #endregion Reset styles */
+
+/* #region General styles */
+html,
+body {
+    height: 100%;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    background-color: #00414f;
+    font-size: 16px;
+}
+
+input {
+    padding-left: 7px;
+}
+
+/* #endregion General styles */
+
+/* #region Wrapper styles */
+.wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+header {
+    flex-grow: 0.5;
+    margin-top: 40px;
+}
+
+main {
+    flex-grow: 1;
+}
+
+form {
+    padding: 30px 40px 40px;
+    border-radius: 10px;
+    background-color: #f3f3f3;
+}
+
+input,
+select {
+    padding: 9px;
+    border-radius: 5px;
+    border: 1px solid rgb(175, 175, 175);
+}
+
+select {
+    cursor: pointer;
+}
+
+input[type="radio"] {
+    cursor: pointer;
+}
+
+input:focus,
+select:focus {
+    border: 1.5px solid #03718a;
+    outline: none;
+}
+
+h1 {
+    text-align: center;
+}
+
+h1,
+h2 {
+    color: #fff;
+}
```

Comment:
It's good to see you grouping elements for styling, so each element gets the same styles, but in this case it's not needed because you set the colour of the `h2` on line 102 separately.